.mocharc.js

```diff
@@ -3,5 +3,5 @@
 module.exports = {
     extension: ["js", "jsx", "ts", "tsx"],
     recursive: true,
-    require: ["./test/ts-node", "./test/setup", "./test/assert-ext", "jsdom-global/register"],
+    require: ["./test/ts-node", "./test/setup", "./test/assert-ext", "./test/configure-testing-library"],
```

Comment:
I would put all of these setup files in extra folder: "test/setup/*", for example.